atom/common/asar/archive.cc

```diff
@@ -107,6 +107,8 @@ bool FillFileInfoWithNode(Archive::FileInfo* info,
     return false;
   info->offset += header_size;
 
+  node->GetBoolean("executable", &info->executable);
```

Comment:
I see that other properties within the same function terminate it with `true` or `false` in some cases I don't fully understand. Can someone explain this?